Mathlib/RingTheory/Ideal/Maps.lean

```diff
@@ -125,11 +125,21 @@ theorem comap_comap {T : Type*} [Semiring T] {I : Ideal T} (f : R →+* S) (g :
     (I.comap g).comap f = I.comap (g.comp f) :=
   rfl
 
+lemma comap_comapₐ {R A B C : Type*} [CommSemiring R] [Semiring A] [Algebra R A] [Semiring B]
```

Comment:
Isn't `R` already in scope?

Reply:
Yes, but it is only a `Semiring`, but I need `CommSemiring` for `Algebra R A` to make sense.